pmprophet/model.py

```diff
@@ -160,6 +243,25 @@ def finalize_model(self):
             pm.Deterministic('y_hat_%s' % self.name, self.y)
 
     def fit(self, draws=500, method='NUTS', map_initialization=False, finalize=True, step_kwargs={}, sample_kwargs={}):
+        """Fit the PMProphet model.
+
+        Parameters
+        ----------
+        draws : Integer, if greater than 0, will do full Bayesian inference
```

Comment:
`If 0, will do MAP estimation.` we don't do this (by design I don't want to support MAP inference when we have all the amazing sampling possibilities with pymc3)